### DISCUSSION TITLE
The difference between two methods "gn_model_analytic" and "ggn_spectrally_separated"

### DISCUSSION THREAD

**Discussion by oolongkiller** [Telecominfraproject/oopt-gnpy#501] [Q&A] [answered]
Hi Team,

I find that there are two methods "gn_model_analytic" and "ggn_spectrally_separated" for computing NLI. What is the difference between them?

The former seems to have higher computing efficiency and higher output "SNR NLI", which has some influences on output "GSNR".

Which of them should I believe?

Thanks,
oolongkiller

**Comment by AndreaDAmico** [comment, marked as answer]
Dear @oolongkiller,
Long story short:
- the "gn_model_analytic" is very fast but less accurate when there is a non negligible power transfer induced by the Raman effect.
-  the "ggn_spectrally_separated" is an accurate model that properly includes the Raman power transfer in the NLI calculation, but it requires a significantly longer computation time.

Therefore, the "ggn_spectrally_separated" is more reliable. Some transmission scenarios, where the Raman effect can be neglected, don't require such accuracy and the "gn_model_analytic" is preferable, e.g. when the C-band is not fully occupied or the transmitted power is limited. 

Best,
Andrea
